examples/01_aeroelasticse/OpenFAST_models/IEA-15-240-RWT/IEA-15-240-RWT-Monopile/IEA-15-240-RWT-Monopile_ElastoDyn.dat

```diff
@@ -53,16 +53,16 @@ True                   PtfmYDOF    - Platform yaw rotation DOF (flag)
 0.0                    UndSling    - Undersling length [distance from teeter pin to the rotor apex] (meters) [unused for 3 blades]
 0.0                    Delta3      - Delta-3 angle for teetering rotors (degrees) [unused for 3 blades]
 0.0                    AzimB1Up    - Azimuth value to use for I/O when blade 1 points up (degrees)
--12.0313               OverHang    - Distance from yaw axis to rotor apex [3 blades] or teeter pin [2 blades] (meters)
+-12.097571763912535    OverHang    - Distance from yaw axis to rotor apex [3 blades] or teeter pin [2 blades] (meters)
 0.0                    ShftGagL    - Distance from rotor apex [3 blades] or teeter pin [2 blades] to shaft strain gages [positive for upwind rotors] (meters)
 -6.0                   ShftTilt    - Rotor shaft tilt angle (degrees)
--4.720                 NacCMxn     - Downwind distance from the tower-top to the nacelle CM (meters)
-0.0                    NacCMyn     - Lateral  distance from the tower-top to the nacelle CM (meters)
-4.275                  NacCMzn     - Vertical distance from the tower-top to the nacelle CM (meters)
--4.985                 NcIMUxn     - Downwind distance from the tower-top to the nacelle IMU (meters)
+-4.7201211961328635    NacCMxn     - Downwind distance from the tower-top to the nacelle CM (meters)
+-0.14642963169858972   NacCMyn     - Lateral  distance from the tower-top to the nacelle CM (meters)
```

Comment:
Can we set this back to 0.0.  The 0.146 value is just a numerical byproduct of WISDEM and some on the ancillary nacelle components, such as power electronics.